Moli/API/UserAPI/MySubPath.swift

```diff
@@ -0,0 +1,47 @@
+import Foundation
+import UIKit
+
+extension UserAPI {
+    static func uploadBannerImage(image: UIImage) async throws {
+        guard let data = image.jpegData(compressionQuality: 1.0) else {
+            print("Unable to get image data in uploadBannerImage")
+            return
+        }
+        let formdataRequest = MultipartFormDataRequest(urlComponents: userURL(subPath: "/my/profile/banner_image"))
+        formdataRequest.addDataField(
+            named: "banner_image",
+            filename: "wallaspace.jpg",
+            data: data,
+            mimeType: "img/jpeg"
+        )
+        return try await APICaller.shared.performRequest(formDataRequest: formdataRequest)
+    }
+
+    static func removeBannerImage() async throws {
+        try await APICaller.shared.performRequest(
+            urlComponents: userURL(subPath: "/my/profile/banner_image"),
+            method: .DELETE)
+    }
+
+    static func uploadProfilePicture(image: UIImage) async throws {
+        guard let data = image.jpegData(compressionQuality: 1.0) else {
+            print("Unable to get image data in uploadBannerImage")
+            return
+        }
+
+        let formdataRequest = MultipartFormDataRequest(urlComponents: userURL(subPath: "/my/profile/image"))
+        formdataRequest.addDataField(
+            named: "profile_image",
+            filename: "wallaspace.jpg",
```

Comment:
:art: Samma som banner, borde kanske vara något mer professionellt? xD
```suggestion
            filename: "profile_image.jpg",
```